README.md

```diff
@@ -1,24 +1,97 @@
-# Astro Docs <img width="19.2" height="25.6" src="https://raw.githubusercontent.com/withastro/astro/main/assets/brand/logo.svg" alt="Astro logo">
+# Astro Docs <img align="right" valign="center" height="52" width="59" src="https://raw.githubusercontent.com/withastro/astro/main/assets/brand/logo.svg" alt="Astro logo" />
```

Comment:
I think this is wrong and it's looking squished. If you want the width to be 59 then the height should be 78.6

Reply:
Great catch. That was a typo. The width is only meant to compliment the height and use the computed value. This is now fixed.